ConwaysGameOfLife.Tests/InputParserShould.cs

```diff
@@ -6,69 +6,45 @@ namespace ConwaysGameOfLife.Tests
 {
     public class InputParserShould
     {
-        InputParser _parser;
-        public InputParserShould()
-        {
-            _parser = new InputParser();
-        }
+        
         [Theory]
         [InlineData("0,0", 0, 0)]
-        [InlineData("-1,-1", -1, -1)]
         [InlineData("100,8", 100, 8)]
-        [InlineData("a,0", -1, 0)]
-        [InlineData("p,s", -1, -1)]
-        [InlineData("3,p", 3, -1)]
-        [InlineData("*,2", -1, 2)]
         public void ReturnLocation_GivenAStringOfOneLocation(string input, int x, int y) 
         {
             var expected = new Location(x, y);
 
-            var result = _parser.ParseLocation(input);
+            var result = InputParser.ParseLocation(input);
 
             Assert.True(UniverseHelper.LocationsAreEqual(expected, result));
         }
 
         [Fact]
-        public void ReturnLocations_GivenAStringOfLocations()
+        public void NotReturnNewLocation_FromInvalidString()

```

Comment:
```suggestion
        public void NotReturnNewLocation_GivenInvalidString()

```
For consistency with the rest of your test names

Reply:
Can we have a few more test cases here e.g. what if there is no comma?